### ISSUE TITLE
7A-8.000MAAE-T is obsolete

### ISSUE THREAD

**Issue by MorganTL** [sinara-hw/Stabilizer#118]
ECS-80-12-30-JGM-TR seems like a good alternative

**Comment by gkasprow** [comment]
does it have same load capacitance?

**Comment by MorganTL** [comment]
> does it have same load capacitance?

Yes, both crystals have the same 12pF load capacitance. Just that ECS-80-12-30-JGM-TR has a higher ESR (80 vs 100ohm).
